### ISSUE TITLE
Option to disable opening file with TSQL query while doing Select Top 1000

### ISSUE THREAD

**Issue by CIRCULARKA** [microsoft/vscode-mssql#17224]
When I click on my table in objects explorer and choose ```Select Top 1000``` option, I end up with data that I need on the right split and useless file with raw TSQL that was used while processing my request that opens in new editor. How can I disable such behaviour so I can just get the table with data withoud raw TSQL file?
![image](https://user-images.githubusercontent.com/34581740/151567764-33e56f20-4d33-418e-aac8-8d792b488d6a.png)

Basically I would like to see something like "mssql.showRawSql: true | false" option

**Comment by abist** [comment]
@CIRCULARKA Currently, this is not supported, but I've marked this as an enhancement.

**Comment by kburtram** [comment]
Thanks for this suggestion! I'm closing older feature requests without significant upvotes as part of routine backlog clean-up.

The functionality described here is similar to what we're planning with the "Data Preview" aspect of the Table Explorer feature, and should be available in a future release.  But this would be separate from the "SELECT 1000" command.
